### PR TITLE
fix #52174  voelkner.de

### LINKED ISSUE #52174: voelkner.de
### Issue URL (Annoyance)
 [https://www.voelkner.de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.voelkner.de%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | uBlock Origin 1.25.2
Browser: | Firefox
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>EasyList, <br/>Peter Lowe's Blocklist<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Language-specific:</b><br/> AdGuard German, <br/>EasyList Germany
Other extensions used: | uMatrix, Cookie AutoDelete

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#52174

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -56,8 +56,8 @@ sallys-blog.de,kaiusa.com,hussel.de,dyson.com.tr,cherry.de,ba-computer.at,stoert
 ! ###gdpr-cookie-message
 demirorenpetrol.com.tr,onurmarket.com,mediamarkt.es,veren.org,yunusonline.com,lightnovelworld.com,burgerking.hu,sparktoro.com,oem-parts.hu,teamspeak.com###gdpr-cookie-message
 !
-digitalo.de#$?#body > .reveal-overlay:has(> #cc_info) { display: none !important; }
-digitalo.de#$#.is-reveal-open { overflow: visible !important; }
+voelkner.de,digitalo.de#?#body > .reveal-overlay:has(> #cc_info)
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_specific.txt]: @BlazDT if I'm not wrong ext-CSS rule is enough here because it don't work in content blockers

**Review by AdamWr — APPROVED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (BlazDT) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Without overflow rule in digitalo.de scrolling was not possible, therefore In added it to it.

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/cookies_specific.txt]: I mean we don't have to add `{ display: none !important; }` to the .reveal-overlay rule =)

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/cookies_specific.txt]: I mean we don't have to add { display: none !important; } to the .reveal-overlay rule =) I've already changed it in commit.

Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt]: yep

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**